deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml

```diff
@@ -96,6 +99,7 @@ spec:
               type: string
           required:
             - PDBForceDrainTimeout
+            - capacityReservation
```

Comment:
Given this will be a mandatory field, have you / can we test what will happen if a cluster has an existing `UpgradeConfig` CR without this field in it, when OLM updates the operator/CRD on a cluster to this version? (before the operator/upgrade config manager even starts up).

Reply:
Hmm, it will get error like this immediately when the CRD updated with new value
```
"error":"UpgradeConfig.upgrade.managed.openshift.io \"managed-upgrade-config\" is invalid: spec.capacityReservation: Required value"```